prac_05/wimbledon.py

```diff
@@ -0,0 +1,51 @@
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual:  60 minutes
+"""
+FILENAME = "wimbledon.csv"
+INDEX_COUNTRY = 1
+INDEX_CHAMPION = 2
+
+
+def main():
+    """Process data from a file and format accordingly."""
+    records = get_records(FILENAME)
+    champion_to_count, countries = process_records(records)
```

Comment:
Would the dictionary name champion_to_count be easier to understand if it was something like "champion_to_number_of_wins"? (I understand the solutions for wimbledon.py also used champion_to_count)

Reply:
Yep 😄